v2/yarpcjson/inbound_test.go

```diff
@@ -149,11 +149,10 @@ func TestHandleBothResponseError(t *testing.T) {
 	}
 	reqBuf := yarpc.NewBufferString(`{"name": "foo", "attributes": {"bar": 42}}`)
 	p := Procedure("simpleProcedure", h)[0]
-	_, resBuf, err := handleWithCodec(context.Background(), req, reqBuf, p)
-
-	require.Equal(t, errors.New("bar"), err)
+	res, resBuf, err := handleWithCodec(context.Background(), req, reqBuf, p)
 
-	var response simpleResponse
-	require.NoError(t, json.Unmarshal(resBuf.Bytes(), &response))
-	assert.Equal(t, simpleResponse{Success: true}, response)
+	require.NoError(t, err)
+	require.NotNil(t, res)
```

Comment:
Should we be asserting something more confidently here, as opposed to not `nil`?